src/domain/entities/note.ts

```diff
@@ -82,3 +82,25 @@ export interface Note {
  * Part of note entity used to create new note
  */
 export type NoteCreationAttributes = Pick<Note, 'publicId' | 'content' | 'creatorId' | 'tools'>;
+
+/**
+ * This type represents only note's content
+ * Used for work with just note's content in web part
+ * This type would be used as a part of the structure, that represents all parent notes of the current note
+ */
+export type NoteIdAndContent = {
```

Comment:
I see no reason to create the new structure for this. We already have `NotePublic` and `NoteList` that could be used here

Reply:
if think that return all notes with full content is a bit heavy operation

Reply:
the content is also returned in this solution. At least, we would need it to compose a note titles.

Reply:
Wouldn't be better to return only the `publicId` and the the `title` of the note? instead of returning the whole content

Reply:
Note has no title as a separate property. It is [computed](https://github.com/codex-team/notes.web/blob/main/src/infrastructure/utils/note.ts#L7) on a client based on its content (editor blocks) — it uses the first text-like block. 

We can think about computing it dynamically on a backend side. But in this case you'll need to get the whole content from the db again and then compute a title. So the profit here is just in reducing of a size of a sending data. Are you sure that this is a real problem?